packages/ui/VirtualList/VirtualList.module.less

```diff
@@ -4,23 +4,32 @@
 	width: 100%;
 	overflow: hidden;
 
-	&.vertical {
+	.content {
+		will-change: transform;
+	}
+}
+
+.vertical {
+	&.native {
 		overflow-y: scroll;
 	}
+	.listItem {
+		width: 100%;
+	}
+}
 
-	&.horizontal {
+.horizontal {
+	&.native {
 		overflow-x: scroll;
 	}
-
-	.content {
-		will-change: transform;
+	.listItem {
+		height: 100%;
 	}
 }
 
 .listItem {
 	position: absolute;
 	overflow: hidden;
-	width: 100%;
 
 	will-change: transform;
```

Comment:
I'm not sure that it is in the scope of this PR, but is the `will-change` needed for native list?

Reply:
Let's discus it in a separate issue.

Reply:
okay